qcodes/instrument_drivers/Attocube/__init__.py

```diff
@@ -0,0 +1 @@
+from .devices import VoltageDivider
```

Comment:
This relative import is wrong and as far as I can see causing travis to fail. It'd be better to import from `qcodes.instrument_drivers.devices`.